Tests/scripts/infrastructure_tests/test_collect_tests_and_content_packs.py

```diff
@@ -1196,3 +1196,13 @@ def test_remove_tests_for_non_supported_packs(tests_to_filter, should_test_conte
         logging.debug.assert_called_once_with(
             'The following test playbooks are not supported and will not be tested: \n{} '.format(
                 '\n'.join(filtered_tests)))
+
+
+@pytest.mark.parametrize('files_string, expected_result', [
```

Comment:
Add a case with each of the files ending - jpg, mp4 and png to make sure the test will catch any change of behavior